website/content/docs/agent/options.mdx

```diff
@@ -1636,6 +1636,8 @@ bind_addr = "{{ GetPrivateInterfaces | include \"network\" \"10.0.0.0/8\" | attr
 
 - `leave_on_terminate` If enabled, when the agent receives a TERM signal, it will send a `Leave` message to the rest of the cluster and gracefully leave. The default behavior for this feature varies based on whether or not the agent is running as a client or a server (prior to Consul 0.7 the default value was unconditionally set to `false`). On agents in client-mode, this defaults to `true` and for agents in server-mode, this defaults to `false`.
 
+- `license_path` <EnterpriseAlert inline /> This specifies the path to a file that contains the Consul Enterprise license. See the [licensing documentation](/docs/enterprise#licensing) for more information about Consul Enterprise license management.
```

Comment:
I would add a sentence that either this or the env vars are required to start Consul enterprise agents after 1.10.

Reply:
The problem here is that for client agents in some circumstances neither this nor the env is required. That is why I preferred a link out to the other licensing documentation which goes over all those details rather than duplicating them here.

So maybe I should add something like. 

```
Alternatively, the license may also be specified by the `CONSUL_LICENSE` or `CONSUL_LICENSE_PATH` environment variables.
```